docs/development/adding-a-package-manager.md

```diff
@@ -67,3 +79,23 @@ This is used when more than one package manager shares settings from a common la
 ### `supportsLockFileMaintenance` (optional)
 
 Set to true if this package manager needs to update lock files in addition to package files.
+
+### `updateArtifacts` (async, optional)
+
+This function is used to run binaries which in turn then will update files.
+Mostly the functionality is used to indirectly update lock files.
+
+In case of directly updating dependencies in lock files use `updateLockedDependency` instead.
+
+`updateArtifacts` gets triggers:
```

Comment:
`updateArtifacts` get executed on the occasion below. This would flip the causality, right? 🤔

Reply:
> `updateArtifacts` get executed on the occasion below. This would flip the causality, right? 🤔

Do you mean that _something else_ triggers the `updateArtifacts` function, and thus `triggers` is wrong?

How about `gets triggered` to make clear _something else_ triggers the function:

```suggestion
`updateArtifacts` gets triggered:
```